src/Atmos.jl

```diff
@@ -83,14 +155,35 @@ function q_liq_threshold end
 """autoconversion timescale [s]  ∈(1e3, 1e4) """
 function τ_acnv end
 
-"""collision efficiency [-]"""
-function E_col end
+"""threshold between ice and snow"""
+function r_ice_snow end
+
+"""collision efficiency for cloud-rain [-]"""
```

Comment:
```suggestion
"""collision efficiency for cloud-rain [unitless]"""
```
It might be nice to explicitly say `unitless` in these circumstances. Sorry if I led the way with `[ ]` or `[-]` elsewhere.

Reply:
`dimensionless`? Though I prefer the shorter `[-]`...

Reply:
I'm fine with either. `[-]` seems like it could be a placeholder for "to add later" 😬

Reply:
perfect!